src/core/main.cpp

```diff
@@ -820,8 +820,7 @@ int main( int argc, char * * argv )
 		if( autoSaveEnabled &&
 			gui->mainWindow()->getSession() != MainWindow::SessionState::Limited )
 		{
-			gui->mainWindow()->runAutoSave();
-			gui->mainWindow()->autoSaveTimerStart();
+			gui->mainWindow()->autoSave();
```

Comment:
I would expect a never-ending auto-save job to be queued up with low priority running in the background.  Can we confirm this is happening?